libs/database/src/lib/entities/site-report.entity.ts

```diff
@@ -52,6 +54,10 @@ export class SiteReport extends Model<SiteReport> {
     return chainScope(this, "approved") as typeof SiteReport;
   }
 
+  static hasBeenSubmitted() {
```

Comment:
This ended up not being needed, I think go ahead and roll back the changes to this file.

Reply:
reverted